<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pshculator</class>
 <widget class="QMainWindow" name="Pshculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatorMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="5">
     <widget class="QPushButton" name="procentof">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QPushButton" name="solveEquation">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="numDivide">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="x">
      <property name="text">
       <string>x^2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="Minus">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="numFactorial">
      <property name="text">
       <string>n!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="MRC">
      <property name="text">
       <string>MRC</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="mPlus">
      <property name="text">
       <string>M÷</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="numFour">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Clear">
      <property name="text">
       <string>CE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="mDevide">
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="numSix">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="numZero">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="PlusMinus">
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="MMinus">
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="numSqrt">
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="numThree">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="numNine">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="label">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="numOne">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="numEight">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Dot">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="mMultiply">
      <property name="text">
       <string>M×</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="numMultiply">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="numSeven">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="numToDegree">
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="numFive">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="numTwo">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="numPlus">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionLicence"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLicence">
   <property name="text">
    <string>Licence</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>MMinus</tabstop>
  <tabstop>mMultiply</tabstop>
  <tabstop>mPlus</tabstop>
  <tabstop>PlusMinus</tabstop>
  <tabstop>numSeven</tabstop>
  <tabstop>numEight</tabstop>
  <tabstop>numNine</tabstop>
  <tabstop>numMultiply</tabstop>
  <tabstop>numSqrt</tabstop>
  <tabstop>numFour</tabstop>
  <tabstop>numFive</tabstop>
  <tabstop>numSix</tabstop>
  <tabstop>numDivide</tabstop>
  <tabstop>x</tabstop>
  <tabstop>numOne</tabstop>
  <tabstop>numTwo</tabstop>
  <tabstop>numThree</tabstop>
  <tabstop>numPlus</tabstop>
  <tabstop>procentof</tabstop>
  <tabstop>numZero</tabstop>
  <tabstop>Dot</tabstop>
  <tabstop>Clear</tabstop>
  <tabstop>Minus</tabstop>
  <tabstop>numToDegree</tabstop>
  <tabstop>mDevide</tabstop>
  <tabstop>solveEquation</tabstop>
  <tabstop>numFactorial</tabstop>
  <tabstop>MRC</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
